cluster/gce/config-default.sh

```diff
@@ -349,6 +350,27 @@ if [[ ! -z "${NODE_ACCELERATORS}" ]]; then
     fi
 fi
 
+# Arktos specific network and service support is a feature that each
+ # pod is associated to certain network, which has its own DNS service.
+ # By default, this feature is enabled in the dev cluster started by this script.
+DISABLE_NETWORK_SERVICE_SUPPORT="${DISABLE_NETWORK_SERVICE_SUPPORT:-}"
+
+DISABLE_ADMISSION_PLUGINS=${DISABLE_ADMISSION_PLUGINS:-""}
+# check for network service support flags
+if [ -z ${DISABLE_NETWORK_SERVICE_SUPPORT} ]; then # when enabled
+  # kubelet enforces per-network DNS ip in pod
+  FEATURE_GATES="${FEATURE_GATES},MandatoryArktosNetwork=true"
+  # tenant controller automatically creates a default network resource for new tenant
+  if [[ "${NETWORK_PROVIDER:-}" == "mizar" ]]; then
```

Comment:
yes, this should be part of https://github.com/CentaurusInfra/arktos/issues/1279 fix, for now, I suggest to leave here. because this PR need merge with issue 1279 fix together. let's wait issue 1279 fix and then update if pessary.

Reply:
I agree with Sonya.